### PR TITLE
MOVE-2691 adding actuator endpoint and bumping snapshot

### PR DESCRIPTION
**Pull request by jaflaten**

### CODE REVIEW

**Review by FrodeBjerkholt — APPROVED** [review]

File: src/main/java/no/digdir/efmstatisticsclient/config/SecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+@Configuration
+@EnableWebFluxSecurity
+public class SecurityConfig {
+    @Bean
+    public SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http) {
+        return http.authorizeExchange()
+                .pathMatchers("/actuator/**").permitAll()
+                .anyExchange().authenticated()
+                .and().build();
```
Comment (martinwam) [inline src/main/java/no/digdir/efmstatisticsclient/config/SecurityConfig.java:17]: return http.securityMatcher(ServerWebExchangeMatchers.anyExchange())
                .cors().and().csrf().disable()
                .authorizeExchange(c -> c.pathMatchers("/actuator/**").permitAll()
                        .anyExchange().authenticated())
                .httpBasic().and().build();

**Review by martinwam — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -10,13 +10,13 @@
...
     <artifactId>efm-statistics-client</artifactId>
-    <version>1.0.3</version>
+    <version>1.1.0-SNAPSHOT</version>
     <name>efm-statistics-client</name>
     <description>Kafka statistikklient for statusmeldinger i eFormidling</description>
     <properties>
         <java.version>11</java.version>
         <kotlin.version>1.4.21</kotlin.version>
-        <move-common.version>1.5.0-SNAPSHOT</move-common.version>
+        <move-common.version>1.8.0</move-common.version>
```
Comment (martinwam) [inline pom.xml:19]: Er ikke typ 1.15.0 siste?

File: pom.xml [hunk 3]
```diff
@@ -63,10 +63,10 @@
     </distributionManagement>
 
     <dependencies>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
-        </dependency>
+<!--        <dependency>-->
```
Comment (martinwam) [inline pom.xml:66]: Bort?

File: pom.xml [hunk 4]
```diff
@@ -83,9 +83,13 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-webflux</artifactId>
         </dependency>
+<!--        <dependency>-->
```
Comment (martinwam) [inline pom.xml:86]: Bort?

**Review by martinwam — APPROVED** [review]

**Pull request merged**